LICENSE.txt

```diff
@@ -0,0 +1,13 @@
+Modified BSD License
```

Comment:
Agreed. I don't know what the placeholder would be (or if we really need one at all). I just wouldn't necessarily put this without asking permission first.